iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/message_queue.hpp

```diff
@@ -111,9 +111,11 @@ class MessageQueue : public DesignPattern::Creation<MessageQueue, IpcChannelErro
 
   private:
     IpcChannelName_t m_name;
-    struct mq_attr m_attributes;
+    struct mq_attr m_attributes
+    {
+    };
     mqd_t m_mqDescriptor = INVALID_DESCRIPTOR;
-    IpcChannelSide m_channelSide;
+    IpcChannelSide m_channelSide{};
```

Comment:
Does this makes sense? This is an enum class and I have no idea what the default constructor of an enum class variable initialize it to. Here I would suggest writing explicitly:
`IpcChannelSide m_channelSide = IpcChannelSide::CLIENT;`. 

It is later set in the ctor correctly so it does not matter but if someone does not initialize it we have at least a defined value of `CLIENT` in here.